src/App.js

```diff
@@ -16,6 +17,7 @@ function App() {
         <Routes>
           <>
             <Route exact path="/" element={<Home />} />
+            <Route exact path="/adds" element={<Adds />} />
```

Comment:
Nice work on including your component in the routes. However, kindly ensure a path which makes it obvious for the user to know what they are doing. '/adds' might be too general.

Current path:
![image](https://user-images.githubusercontent.com/59999191/197908935-4f70a404-88f3-4ef9-b3bc-756cef45a167.png)

Suggested path:
![image](https://user-images.githubusercontent.com/59999191/197909535-af75d5d7-407c-4fdd-87a2-0f37075654e1.png)

Reply:
Done